app/components/Views/OnboardingSuccess/OnboardingAssetsSettings/index.tsx

```diff
@@ -0,0 +1,73 @@
+// Third party dependencies
+import React, { useCallback, useLayoutEffect } from 'react';
+import { ScrollView, TouchableOpacity } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+// External dependencies
+import Text, {
+  TextVariant,
+} from '../../../../component-library/components/Texts/Text';
+import Icon, {
+  IconSize,
+  IconName,
+} from '../../../../component-library/components/Icons/Icon';
+import { strings } from '../../../../../locales/i18n';
+import { useStyles } from '../../../../component-library/hooks';
+import AutoDetectTokensSettings from '../../Settings/AutoDetectTokensSettings';
+import DisplayNFTMediaSettings from '../../Settings/DisplayNFTMediaSettings';
+import AutoDetectNFTSettings from '../../Settings/AutoDetectNFTSettings';
+import IPFSGatewaySettings from '../../Settings/IPFSGatewaySettings';
+import IncomingTransactionsSettings from '../../Settings/IncomingTransactionsSettings';
+import BatchAccountBalanceSettings from '../../Settings/BatchAccountBalanceSettings';
+
+// Internal dependencies
+import styleSheet from './index.styles';
+
+const AssetSettings = () => {
+  const { styles } = useStyles(styleSheet, {});
+  const navigation = useNavigation();
+
+  const renderBackButton = useCallback(
+    () => (
+      <TouchableOpacity
+        onPress={() => navigation.goBack()}
+        style={styles.backButton}
+      >
+        <Icon name={IconName.ArrowLeft} size={IconSize.Lg} />
+      </TouchableOpacity>
```

Comment:
non-blocking: you could use the standardized `ButtonIcon` here. You could then remove the styles as it already has the necessary padding.

```suggestion
      <ButtonIcon
        size={ButtonIconSizes.Lg}
        iconName={IconName.ArrowLeft}
        accessibilityRole="button"
        accessibilityLabel={} // Add "back" or similar accessibility label
        onPress={() => navigation.goBack()}
      />
```

Reply:
I can address the non-block issues as for breaking up the pull request I totally could have in retrospect. What I had assumed would be a slightly smaller PR turned out larger in the end. With better planning I could have split this into two PRs. One or several PRs for modular settings components and the other new screens. I'll be more mindful next time. thanks @georgewrmarshall for the feedback!

Reply:
Sounds great, @vinnyhoward! I typically start with a comprehensive draft PR that I share with the team for status updates, demos, and initial feedback. This approach allows me to complete all functionalities and decide how to break them down into smaller PRs for detailed reviews. Once I’ve split them, I’ll request reviews and may reference the larger draft PR if needed.

Reply:
I actually am going to do that exact method! Thanks for that idea. I'll use this as my draft reference PR and build off that.